.browserslistrc

```diff
@@ -0,0 +1,18 @@
+[production]
+>0.5% in alt-eu and >0.5% in alt-as and >0.5% in alt-na and >0.5% in alt-sa and >0.5% in alt-af and >0.5% in alt-oc and >1%
+not ie > 0
+last 2 Chrome versions
+last 2 Edge versions
+last 2 Safari versions
+Firefox esr
+
+[supported]
```

Comment:
should we mention firefox ESR here?

Reply:
It depends. Is Firefox "supported" in the sense that we specifically develop for it, or just a nuisance we have to deal with whether we want it or not? I'd say the second. I see the "supported" list as more of a statement about the browsers people are using than the actual reality. Otherwise, we'd also add all the niche ones we're supporting in "production".